app/presenters/publishing_api/ministers_index_presenter.rb

```diff
@@ -51,5 +54,40 @@ def details
     def base_path
       "/government/ministers"
     end
+
+    def reshuffle_in_progress
+      SitewideSetting.find_by(key: :minister_reshuffle_mode)
```

Comment:
could you do `SitewideSetting.find_by(key: :minister_reshuffle_mode) || false` or similar here to avoid needing to `&` try in each of the call sites of this method?

Reply:
Nice idea.  I'll change this.